sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -185,10 +189,11 @@ object Webpack {
 
     log.info("Bundling the application with its NPM dependencies")
     val args = extraArgs ++: Seq("--config", configFile.absolutePath)
-    Webpack.run(args: _*)(targetDir, log)
+    val stats = Webpack.run(args: _*)(targetDir, log)
+    stats.foreach(_.print(log))
 
-    // TODO Support custom webpack config file (the output may be overridden by users)
-    val bundle = generatedWebpackConfigFile.asApplicationBundle
+    // Attempt to discover the actual name produced by webpack indexing by chunk name and discarding maps
+    val bundle = generatedWebpackConfigFile.asApplicationBundle(stats)
     assert(bundle.file.exists(), "Webpack failed to create application bundle")
     bundle
```

Comment:
As mentioned above, I think we should return all the bundles produced by Webpack.